packages/models-library/src/models_library/services_resources.py

```diff
@@ -1,10 +1,28 @@
-from typing import Union
+import logging
+from typing import Any, Union
 
 from models_library.generics import DictModel
-from pydantic import BaseModel, StrictFloat, StrictInt, root_validator
+from pydantic import (
+    BaseModel,
+    ByteSize,
+    Field,
+    StrictFloat,
+    StrictInt,
+    constr,
+    root_validator,
+)
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_RESOURCES: dict[str, Any] = {
+    "CPU": {"limit": 0.1, "reservation": 0.1},
```

Comment:
THOUGHT: why set here default values for resources? Remember that this is a library. A library knowns nothing about the resources of the computer it runs. It sounds to me more reasonable to do this in a service, right?